src/app/static/src/tests/baseline/mutations.test.ts

```diff
@@ -56,23 +57,13 @@ describe("Baseline mutations", () => {
         expect(testState.pjnzError).toBe("Some error");
     });
 
-    it("sets country and filename and clears error if present on PJNZUpdated", () => {
-
-        const testState = {...initialBaselineState, pjnzError: "test"};
```

Comment:
I'm guessing you deleted this because it's a duplicate of https://github.com/mrc-ide/hint/pull/122/files#diff-5fa3e0f1539bd0a346ad87323763551fL9 ? But looks like this one tests that the `pjnzError` gets cleared, whereas the other one doesn't.

Reply:
Ah yeah, I just saw that it was testing the same properties, but of course this is with an error already present...